static/js/lib/currency.js

```diff
@@ -37,7 +37,9 @@ const codesToOptions = R.compose(
 
 export const currencyOptions = codesToOptions(cc.codes());
 
-export const codeToCountryName = (code: string) => iso3166.country(code).name || '';
+export const codeToCountryName = (code: string) =>  R.pathOr(
+  "N/A", ['name'], iso3166.country(R.defaultTo("", code))
+);
```

Comment:
could we make this function return `''` instead of `'N/A'` and only display `'N/A'` in the `countryRefinementOption`? I think that the change to `'N/A'` might mess up the `currencyForCountry` function below.